extension/src/popup/components/warningMessages/FlaggedWarningMessage.tsx

```diff
@@ -16,22 +16,22 @@ const DangerousAccountWarning = ({
   isUnsafe: boolean;
   isMalicious: boolean;
 }) => {
+  if (isMalicious) {
```

Comment:
if account `isMalicious`, show malicious warning message. But only show unsafe warning message if account is not also malicious